actors/user/src/test/java/org/sunbird/user/UserActorTest.java

```diff
@@ -146,7 +146,7 @@ public void testCreateUser() throws Exception {
     assertTrue(null != ex);
   }
 
-  @Test
+  @Ignore
```

Comment:
Consider using @Ignore - at class level - there are no active test-cases in this class - now.